tasks/ldap.yml

```diff
@@ -2,26 +2,26 @@
 # vim: foldmarker=[[[,]]]:foldmethod=marker
 
 - name: Check default ldap config exists
-  command: php --file "{{ owncloud__home }}/occ" ldap:show-config ""
+  command: '{{ owncloud__occ_bin_file_path | quote }} ldap:show-config'
   changed_when: False
   when: (not ansible_check_mode)
   register: owncloud__register_ldap_default_config_exits
   become_user: '{{ owncloud__user }}'
```

Comment:
Yes. I dont know why it dont work. My nginx local facts:

`root@cloud:~# cat /etc/ansible/facts.d/nginx.fact 
#!/bin/bash

# Check nginx version
nginx_version="$(/usr/sbin/nginx -v 2>&1 | sed -e 's#^.*/##')"

cat << EOF
{
"acme": "true",
"acme_domain": "acme.compraactiva.me",
"acme_root": "/srv/www/sites/acme/public",
"acme_server": "false",
"default_server": "cloud.compraactiva.me",
"default_server_ssl": "cloud.compraactiva.me",
"manage_ipv6only": "true",
"user": "www-data",
"www": "/srv/www",
"flavor": "full",
"version": "$nginx_version",
"status_localhost": [
    "127.0.0.1/32", 
    "::1/128", 
    "5.196.121.16"
],
"status": []
}
EOF`

So, i think is better use OCC custom command.